azurelinuxagent/common/osutil/default.py

```diff
@@ -665,9 +665,9 @@ def get_mac_addr(self):
         Convienience function, returns mac addr bound to
```

Comment:
Might as well fix this typo, since you're here anyway. "Convienience" is misspelled.

Reply:
Sure, fixed